### DISCUSSION TITLE
What is default gui port for peering manager docker?

### DISCUSSION THREAD

**Discussion by kronicklez** [peering-manager/peering-manager#896] [Q&A]
Hi all,

Is it the default port for peering manager docker GUI was 8080? I try https://ip_address:8080 but cannot open the page.

Appreciate any help.

![image](https://github.com/user-attachments/assets/6894cf4b-5655-4e06-bed1-f4d4a77f5f5a)

**Comment by gmazoyer** [comment]
This is explained in the [README file](https://github.com/peering-manager/docker?tab=readme-ov-file#running-with-docker-compose) of the docker repository. The port is not exposed by default, you need an override file to do it yourself. [An example](https://github.com/peering-manager/docker/blob/main/docker-compose.override.yml.example) is provided in the repository too.
